ehr/test/src/org/labkey/test/util/ehr/EHRTestHelper.java

```diff
@@ -230,13 +230,29 @@ public void applyTemplate(Ext4GridRef grid, String templateName, boolean bulkEdi
         }
     }
 
+    public Locator.XPathLocator openBulkEdit(Ext4GridRef grid)
+    {
+        grid.clickTbarButton("More Actions");
+        _test.waitAndClick(Locator.tag("span").withText("Bulk Edit"));
+        Locator.XPathLocator bulkEditWindow = Ext4Helper.Locators.window("Bulk Edit");
+        _test.waitForElement(bulkEditWindow);
+        return bulkEditWindow;
+    }
+
     public void toggleBulkEditField(String label)
```

Comment:
Consider a comment that this does a contains check?

Reply:
done